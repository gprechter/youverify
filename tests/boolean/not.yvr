a: BOOL
b: BOOL
c: BOOL

a = true
b = false
a = ! a
b = ! b
c = ! b