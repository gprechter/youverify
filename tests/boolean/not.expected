{
    "a": false,
    "b": true,
    "c": false
}